internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -755,6 +772,16 @@ func (r *BMACReconciler) reconcileSpokeBMH(ctx context.Context, log logrus.Field
 		Namespace: agent.Spec.ClusterDeploymentName.Namespace,
 		Name:      fmt.Sprintf(adminKubeConfigStringTemplate, cd.Name),
 	}
+	isNonePlatform, err := r.isNonePlatform(ctx, cd)
+	if err != nil {
+		log.WithError(err).Error("Is none platform")
+		return reconcileError{err}
```

Comment:
@flaper87 is that reconcile immediately? if so can it create a hot loop?

Reply:
Yeah, the controller will likely reconcile again as soon as possible. If we want to re-queue, then I think using `reconcileRequeue` may be better. We could rework this so that we react to changes on ACI by adding a watch, but that doesn't save us from errors and it may hide important issues.

Reply:
It should be handled by reconcileError wrapper